lib/data/models/browser_tab.dart

```diff
@@ -22,30 +16,14 @@ class BrowserTab with _$BrowserTab {
     /// The url of the tab.
     @uriJsonConverter required Uri url,
 
-    /// The screenshot of the tab.
-    required String? image,
+    /// The screenshot path of the tab.
+    required String? imageId,
```

Comment:
Why `imageId` if this is path ?

Reply:
It is not a path, it's an id, because path will be calculated on the fly from this id, tab id and app docs  path (which can be changed by os from app run to run).